pkg/sql/schemachanger/scerrors/errors.go

```diff
@@ -75,7 +75,7 @@ func (el EventLogger) HandlePanicAndLogError(ctx context.Context, err *error) {
 			log.InfofDepth(ctx, depth, "done %s in %s", el.msg, redact.Safe(timeutil.Since(el.start)))
 		}
 	case HasNotImplemented(*err):
-		log.InfofDepth(ctx, depth, "failed %s with error: %v", el.msg, *err)
+		log.InfofDepth(ctx, depth, "unsuccessful in %s with message: %v", el.msg, *err)
```

Comment:
How about something more straightforward like`declarative schema changer fallback %s with message:`, `unsuccessful` still sounds erroneous heh.

Reply:
Adjusted to get rid of unsuccessful (the way the strings are joined doesn't help):
falling back from building declarative schema change targets for GRANT with message: ‹*tree.Grant not implemented in the new schema changer›

Reply:
Can we just get rid of this log message altogether? I don't think we get a ton of value from it. How about:

```
log.VEventfDepth(ctx, depth, 1, "declarative schema changer does not support %s: %v", el.msg, *err)
```

Reply:
I'll go with that approach.